### PR TITLE
Removed SQL Persistence Query and deprecated PersistentView

### PR DESCRIPTION
**Pull request by alexvaluyskiy**
Removed SQL Persistence Query
Deprecated PersistentView

### CODE REVIEW

File: src/core/Akka.Persistence/PersistentView.cs [hunk 1]
```diff
@@ -1,317 +0,0 @@
...
-    /// to the view with a <see cref="SnapshotOffer"/> message, followed by replayed messages, if any, that are younger
-    /// than the snapshot. Default is to offer the latest saved snapshot.
-    /// 
-    /// By default, a view automatically updates itself with an interval returned by `autoUpdateInterval`.
-    /// This method can be overridden by implementation classes to define a view instance-specific update
-    /// interval. The default update interval for all views of an actor system can be configured with the
-    /// `akka.persistence.view.auto-update-interval` configuration key. Applications may trigger additional
-    /// view updates by sending the view <see cref="Update"/> requests. See also methods
-    /// </summary>
-    public abstract partial class PersistentView : ActorBase, ISnapshotter, IPersistentIdentity, IWithUnboundedStash, IPersistenceRecovery
```
Comment (alexvaluyskiy) [inline src/core/Akka.Persistence/PersistentView.cs]: `ISnapshotter` interface was implemented here and should be implemented in `Eventsourced` class (but not in our case). If we remove `PersistentView`, should we remove this useless interface too?

Comment (ismaelhamed) [inline src/core/Akka.Persistence/PersistentView.cs]: Was this advertised anywhere? This is a breaking change that will prevent us from upgrading to v1.3 right away.

Comment (alexvaluyskiy) [inline src/core/Akka.Persistence/PersistentView.cs]: We mention it in the documentation one year ago: http://getakka.net/docs/persistence/persistent-views

Comment (alexvaluyskiy) [inline src/core/Akka.Persistence/PersistentView.cs]: It is still discussable. I've created this meta-issue https://github.com/akkadotnet/akka.net/issues/2743 to discuss all things

**Comment by alexvaluyskiy** [comment]
@ismaelhamed now PersistentView restored and deprecated

**Comment by ismaelhamed** [comment]
> PersistentView was deprecated and will be removed in the next major version [1.3.0]

Good. But, did you mean version 1.4.0?

**Comment by alexvaluyskiy** [comment]
@ismaelhamed We don't have any date for 1.4 yet. First of all, we should release Persistence Query support for all popular Persistence providers, then to check them in production. After that, we could decide to remove PersistenceView.
It is our fault that we did not deprecate `PersistenceView` before.

**Pull request merged**